tests/unit/backend/gene_info/test_v1.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import requests
+import json
+
+from backend.gene_info.api import ncbi_provider
+from tests.unit.backend.corpora.fixtures.environment_setup import EnvironmentSetup
+from backend.corpora.api_server.app import app
+import xml.etree.ElementTree as ET
+from unittest.mock import patch, call
+
+
+class GeneInfoAPIv1Tests(unittest.TestCase):
+    """Tests Gene Info API endpoint"""
+
+    def setUp(self):
+        super().setUp()
+        with EnvironmentSetup(dict(APP_NAME="corpora-api")):
+            self.app = app.test_client(use_cookies=False)
+        self.final_gene_info_result = {
+            "name": "",
+            "summary": "",
+            "ncbi_url": "https://www.ncbi.nlm.nih.gov/gene/1",
+            "synonyms": [],
+        }
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.maxDiff = None
+
+    @patch("backend.gene_info.api.ncbi_provider.urllib.request.urlopen")
```

Comment:
Mock can be a tough concept to grasp and figuring out where to patch is even tough. Great job with this.